integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -17,7 +18,79 @@ All examples in this starter kit require that you use the [new IBM watsonx Assis
+- `model_parameters_max_new_tokens` : The maximum number of new tokens to be generated. Defaults to 300.
+- `model_parameters_min_new_tokens`: The minimum number of the new tokens to be generated. Defaults to 1.
+- `model_parameters_repetition_penalty`: Represents the penalty for penalizing tokens that have already been generated or belong to the context. The range is 1.0 to 2.0 and defaults to 1.1.
+- `model_parameters_stop_sequences`: Stop sequences are one or more strings which will cause the text generation to stop if/when they are produced as part of the output. Stop sequences encountered prior to the minimum number of tokens being generated will be ignored. The list may contain up to 6 strings. Defaults to [].
+  _Note_: For some models, you may want to set this to ["\n\n"] (i.e., stop whenever the model produces a double newline) because that can be a reliable indication that the model is shifting to a new topic. However, that is generally unhelpful for chat models like the one that we're using in this kit, because those models are unlikely to suddenly shift to a new topic. The model we're using in this kit does often include double newlines within a coherent answer, so we don't recommend that setting with this model.
+- `model_parameters_temperature` : The value used to control the next token probabilities. The range is from 0 to 1.00; 0 makes it deterministic.
+- `model_response`: The text generated by the model in response to the model input.
+- `passages` : Concatenation of top search results.
+- `query_body`: The query body that will be sent to Elasticsearch search index. The query body is set and used by the Search action.
+- `query_source`: The query source that will be sent to Elasticsearch search index. The query source is set and used by the Search action.
+- `query_text`: You MAY change this to pass queries to Watson Discovery. By default the Search action passes the user’s input.text directly.
+- `search_results`: Response object from Elasticsearch search query. It is set and used by the Search action.
+- `snippet` : Top results from the Watson Discovery document search.
+- `verbose`: A boolean that will print debugging output if true. Default is false. Note that if you turn this on, you will see the prompt we send to the model rendered as if it were HTML, which looks messy (all the text is crossed out because the start of the prompt is a tag that Llama uses to represent the start of a message but represents strike-through in HTML).
+- `watsonx_api_version` - watsonx api date version. It currently defaults to `2023-05-29`.
+- `watsonx_project_id`: You **MUST** set this value to be [a project ID value from watsonx](https://dataplatform.cloud.ibm.com/docs/content/wsj/manage-data/manage-projects.html). By default, this is a [sandbox project id](https://dataplatform.cloud.ibm.com/docs/content/wsj/manage-data/sandbox.html) that is automatically created for you when you sign up for watsonx.ai.
```

Comment:
`watsonx_project_id` is set to some value in the Actions json file.  It should not be set in the json file -- the user needs to set it.

Also, the Discovery->watsonx kit has a very nice check to see if `watsonx_project_id` is set.  If it is not, it provides instructions for doing so.  I would recommend following that approach.

Reply:
I removed the default value for `watsonx_project_id` in the workspace JSON file. I couldn't find what you said about ```the Discovery->watsonx kit has a very nice check to see if watsonx_project_id is set```. But without the default value, any call to the watsonx extension would fail with a message saying that project_id should be set.